pkg/cli/flags.go

```diff
@@ -0,0 +1,31 @@
+package cli
```

Comment:
I could move them into cmd/lassie. I think this might have been left over from a refactor. I moved those directories around like 20 times before settling on something.